capture-sdk/sdk/src/main/java/net/gini/android/capture/Amount.java

```diff
@@ -0,0 +1,35 @@
+package net.gini.android.capture;
+
+import androidx.annotation.NonNull;
+
+import java.math.BigDecimal;
+
+public class Amount {
+
+    private static Amount mAmount;
```

Comment:
[Suggestion] What about simply creating a public static final property like this (or a companion object property with `@JvmStatic` annotation in Kotlin):
```
public static final Amount EMPTY = new Amount(BigDecimal.valueOf(0), AmountCurrency.EUR);
```

This way it's less code and also [thread safe](https://stackoverflow.com/a/25868299).

Reply:
Okay totally makes sense. Thanks